acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -162,7 +162,7 @@ def install_repos_on(host, project, sha, repo_configs_dir)
                     :puppet_agent_version => ENV['SUITE_VERSION'] || ENV['SHA']
                 }
                 # this installs puppet-agent on windows (msi) and osx (dmg)
-                install_puppet_agent_dev_repo_on(agent, opts)
+                install_from_build_data_url('puppet-agent', "http://builds.delivery.puppetlabs.net/puppet-agent/#{opts[:puppet_agent_sha]}/artifacts/#{opts[:puppet_agent_sha]}.yaml")
```

Comment:
This removes the need for the `opts` object completely. Just use `ENV['SHA']` directly.

Reply:
Actually, `install_repos_on` is never used, so we could remove it completely.

Reply:
Same is true of everything else in this file except `install_packages_on`.

Reply:
Should I just clean up this file?